src/wp-includes/taxonomy.php

```diff
@@ -231,8 +231,25 @@ function create_initial_taxonomies() {
 			'publicly_queryable' => false,
 			'hierarchical'       => false,
 			'labels'             => array(
-				'name'          => _x( 'Pattern Categories', 'taxonomy general name' ),
-				'singular_name' => _x( 'Pattern Category', 'taxonomy singular name' ),
+				'name'                       => _x( 'Pattern Categories', 'taxonomy general name' ),
+				'singular_name'              => _x( 'Pattern Category', 'taxonomy singular name' ),
+				'add_new_item'               => __( 'Add New Category' ),
+				'add_or_remove_items'        => __( 'Add or remove pattern categories' ),
+				'back_to_items'              => __( '&larr; Go to pattern categories' ),
+				'choose_from_most_used'      => __( 'Choose from the most used pattern categories' ),
+				'edit_item'                  => __( 'Edit Pattern Category' ),
+				'item_link'                  => __( 'Pattern Category Link' ),
+				'item_link_description'      => __( 'A link to a pattern category.' ),
+				'items_list'                 => __( 'Pattern Categories list' ),
+				'items_list_navigation'      => __( 'Pattern Categories list navigation' ),
+				'new_item_name'              => __( 'New Pattern Category Name' ),
+				'no_terms'                   => __( 'No pattern categories' ),
+				'not_found'                  => __( 'No pattern categories found.' ),
+				'popular_items'              => __( 'Popular Pattern Categories' ),
+				'search_items'               => __( 'Search Pattern Categories' ),
+				'separate_items_with_commas' => __( 'Separate pattern categories with commas' ),
```

Comment:
For "pattern categories," we use different text cases that need to be consistent.

Reply:
`separate_items_with_commas` is lowercase for regular tags by default, so this seems correct.

Reply:
Yes, I did copy from the tags I think and adapted the labels.

Reply:
For Link Category taxonomy we use `'back_to_items' => __( '&larr; Go to Link Categories' ),` and for Pattern Category we use `'back_to_items' => __( '&larr; Go to pattern categories' ),` it should be `'back_to_items' => __( '&larr; Go to Pattern Categories' ),`

Reply:
Here's a follow-up https://github.com/WordPress/wordpress-develop/pull/5930